odoo/addons/base/models/ir_module.py

```diff
@@ -947,6 +947,11 @@ def _get_id(self, name):
         self.env.cr.execute("SELECT id FROM ir_module_module WHERE name=%s", (name,))
         return self.env.cr.fetchone()
 
+    @tools.ormcache('name')
+    def _get_data(self, name):
```

Comment:
Could you please name it like we already do for website and lang module so it is easier to grep and figure stuff?
See `_get_cached()` and `_get_cached_values()`